tests/console/docker.pm

```diff
@@ -116,6 +116,21 @@ sub run {
         assert_script_run('man -P cat docker config | grep "docker-config - Manage Docker configs"');
     }
 
+    # Try to stop container using ctrl+c
+    type_string("docker run --rm opensuse/tumbleweed sleep 99999\n");
+    type_string("# Lets's press ctrl+c now ... ");
+    send_key 'ctrl-c';
+    type_string(" ... and we seem to be still in container\n");
+    save_screenshot();
+    # It does not work by purpose - see bsc#10738771 and bsc#1089732
+    # We need to kill the process from another console
+    select_console('user-console');
+    script_sudo('ps xo "pid flag args label" | grep "sleep 99999"');
```

Comment:
Just a suggestion - it might be safer to sleep only 30 seconds - and let docker run finish by itself, not kill it. It will be easier to maintain - but it's up to you. Suggestion in code:
```perl
type_string("docker run --rm opensuse/tumbleweed sleep 20\n");
type_string("# Lets's press ctrl+c now ... ");
send_key 'ctrl-c';
type_string(" ... and we seem to be still in container\n");

# If script timeouts we know ctrl-c did not work
if (script_run('timeout 10 ls') == 124) {
  sleep 10;
} else {
  die 'ctrl-c stopped container';
}
```

Reply:
I agree with that. Plus one more thing: make the title of the commit smaller.

Reply:
Done, here is the test run for [SLE12sp3](http://pdostal-server.suse.cz/tests/777).

Reply:
So it is not done yet, I'm still having problems to get the desired result.